cmd/kpn/main.go

```diff
@@ -45,37 +45,37 @@ func init() {
 	// Initialize the CLI app and start kpn
 	app.Action = nodecmd.RunKlaytnNode
 	app.HideVersion = true // we have a command to print the version
-	app.Copyright = "Copyright 2018-2023 The klaytn Authors"
-	app.Commands = []cli.Command{
+	app.Copyright = "Copyright 2018-2019 The klaytn Authors"
```

Comment:
why did you change this?

Reply:
Oh, I missed it when resolving conflict. Thanks